web/src/containers/admin/CreateAccount.js

```diff
@@ -23,15 +23,16 @@ class CreateUser extends Component {
   };
 
   toggleShowPassword = () => {
-    this.setState({ showPassword: !this.state.showPassword });
+    this.setState(prev => ({ showPassword: !prev.showPassword }));
```

Comment:
By happenstance, this new rule also caught this bad practice: you shouldn't access `this.state` in a call to `setState` because those calls are queued and run asynchronously.  Eg, if `showPassword` is false, then you had this:

```js
toggleShowPassword();
toggleShowPassword();
```

...you'd then expect `showPassword` to still be false (toggled to true then back to false).  But in reality, both calls to `toggleShowPassword()` capture `this.state` before it has been updated, so they both start with `showPassword` false and change it to `true`, so the end result is that `showPassword` is true.

If you queue up functions instead of new objects, they are always called in the order they were triggered, and they are called with the most recent version of state after all previous updates in the cycle were applied.

Reply:
> If you queue up functions instead of new objects, they are always called in the order they were triggered, and they are called with the most recent version of state after all previous updates in the cycle were applied.

Suddenly, mystery bugs from previous projects make sense...